sonar-plugin-api/src/main/java/org/sonar/api/batch/fs/IndexedFile.java

```diff
@@ -43,6 +44,7 @@
    * <br>
    * Relative path is not null and is normalized ('foo/../foo' is replaced by 'foo').
    */
+  @Deprecated
   @Override
   String relativePath();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)